app/content/views/user.py

```diff
@@ -156,9 +154,18 @@ def get_user_events(self, request, *args, **kwargs):
             for registration in registrations
             if not registration.event.expired
         ]
-        page = self.paginate_queryset(events)
-        serializer = EventListSerializer(page, many=True)
-        return self.get_paginated_response(serializer.data)
+
+        return self.paginate_response(data=events, serializer=EventListSerializer)
+
+    @action(detail=False, methods=["get"], url_path="me/forms")
+    def get_user_forms(self, request, *args, **kwargs):
+        forms = request.user.forms
+
+        filter_unanswered = request.query_params.get("unanswered")
+        if filter_unanswered:
```

Comment:
Her trengs det en sjekk på at `unanswered is True` ettersom `/me/forms/?unanswered=false` blir True her.

![image](https://user-images.githubusercontent.com/31009729/134046912-4f7f2e29-f292-4db1-bb3a-0439ab914409.png)
![image](https://user-images.githubusercontent.com/31009729/134046942-8a5b0874-3875-4b20-b4f5-16220217f8e2.png)

Reply:
Måtte bli litt kreativ her siden DRF ikke håndterer boolske verdier i query params så bra 😊